### ISSUE TITLE
EPIP: Adapt 5.5 (and eventually 5.6) to the new part 6

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx#273]
With EPIAP the information in EPIP about accessibility should be reduced to the minimum and mostly contain a reference to part 6.
At least a check for conformance must be made.

**Comment by ue71603** [comment]
https://github.com/NeTEx-CEN/NeTEx-Profile-EPIP/issues/3
